lib/action_policy/testing.rb

```diff
@@ -34,21 +34,23 @@ def context_matches?(context, actual)
       end
 
       class Scoping # :nodoc:
-        attr_reader :policy, :target, :type, :name, :scope_options
+        attr_reader :policy, :target, :type, :name, :scope_options, :context
 
         def initialize(policy, target, type, name, scope_options)
           @policy = policy
           @target = target
           @type = type
           @name = name
           @scope_options = scope_options
+          @context = policy.authorization_context
```

Comment:
We already have `policy` in the state, no need to extract context into its own instance variable.